apps/src/p5lab/utils.js

```diff
@@ -17,3 +17,33 @@ export function arrayEquals(a, b) {
 
   return true;
 }
+
+export function capitalizeFirstLetter(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+export function parsePathString(text) {
+  // Example string paths:
+  // 'sound://category_board_games/card_dealing_multiple.mp3'
+  // 'sound://default.mp3'
+  if (!text.startsWith('sound://') || !text.endsWith('.mp3')) {
+    throw new Error('This is not a valid path to a sound file.');
```

Comment:
What's the expected behavior if this happens? looks like currently this error would go unhandled and crash the page - but maybe we want something to break and indicate to the user that we're in an invalid state?

Reply:
Good point! I think that I'll go ahead and include this check in the sound picker `validation` function as a more general check than just checking for 'Choose`. So if the sound value is invalid, then the default sound value will be assigned. I also added a `console.error` log. Lmk what you think!

<img width="1125" alt="Screen Shot 2023-05-04 at 10 42 20 AM" src="https://user-images.githubusercontent.com/107423305/236258984-7b13fb34-0c92-4fe9-b113-44a6bfc6b66f.png">

Reply:
Sounds good! do we still need this error here then? I suppose reaching this error would mean we've hit an even stranger case where the text has somehow been changed after validation

Reply:
Yeah, I'm placed it here just in case we want to use in another lab like Game Lab which uses sounds.